### ISSUE TITLE
Revamp repo and its purpose

### ISSUE THREAD

**Issue by valentina-s** [orcasound/orcadata#32]
I find the purpose of this repo kind of confusing.  The name and the access.md file are focused on `orca data`. However, the description focuses and other folder focuses on software. The main focus seems on preprocessing software, since a lot of the more advanced software now is in other repos, so the scope can be updated. 

I also find the [diagram](https://camo.githubusercontent.com/fb903194065306e0201bd13408c02a60ae315d20da920f1dd4571f63cca345ac/687474703a2f2f6f726361736f756e642e6e65742f696d672f6f726361736f756e642d6170702f4f726361736f756e642d736f6674776172652d65766f6c7574696f6e2d6d6f64656c2e706e67)  in the access.md very hard to read since it points leftwards! It most probably needs a caption, and there may be a better place for it, unless one has a better way to relate it to the access topic.

Maybe with some community suggestions/discussion the organization of this repo can be improved.

**Comment by paulcretu** [comment]
I agree @valentina-s, the original purpose of this repo was to provide access to the AWS data (instructions and software), kind of what `orca-hls-utils` is. Over time it became a catch-all for various data processing experiments. We should either clean it up or archive it. I'm not exactly sure what it should be—I like the idea of having a single go-to repo for figuring out how to access the data.

Perhaps revamping the repo could be a part of https://github.com/orcasound/orcagsoc/issues/32? If `orca-hls-utils` is improved, it might be good to move it into here (as the scope will expand beyond HLS to include FLAC data as well, and eventually metadata).

Or maybe it's fine to leave `orca-hls-utils` where it is, but make orcadata itself into a package that acts as an entrypoint for accessing data—basically some sort of high-level orcasound CLI. I always imagined doing something like:

```
> orcadata list nodes
> orcadata list audio --node <node_name>
> orcadata get audio --node <node_name> --format hls <start_date> <end_date>
> orcadata get detections --node <node_name> --format csv <start_date> <end_date>
> orcadata get ais --node <node_name> --format csv <start_date> <end_date>
```

as well as having an SDK so other tools can automatically use the commands programmatically. We could even hook up data processing commands:

```
> orcadata spectrogram --node <node_name> <start_date> <end_date>
> orcadata convert hls-to-wav <directory>
```

**Comment by scottveirs** [comment]
In today's standup, @valentina-s announced/demoed that she had fixed the right-to-left diagram by making it "read" left-to-right!
